au/code/au/utility/test/string_constant_test.cc

```diff
@@ -50,6 +64,13 @@ TEST(IToA, ValueHoldsStringVersionOfTemplateParameter) {
 
     EXPECT_STREQ(IToA<-1>::value.c_str(), "-1");
     EXPECT_STREQ(IToA<-83294>::value.c_str(), "-83294");
+
+    // This funny way of writing it is because `-9'223'372'036'854'775'808` isn't a literal.  The
+    // actual literal is `9'223'372'036'854'775'808`, which is too big (by one) to fit into
+    // `int64_t` --- even though `-9'223'372'036'854'775'808` can!  So instead, we negate the
+    // largest literal, and then subtract one to get the lowest literal.
+    constexpr int64_t min = -9'223'372'036'854'775'807 - 1;
+    EXPECT_STREQ(IToA<min>::value.c_str(), "-9223372036854775808");
```

Comment:
Do we want to start using the `StrEq` matcher for things like this?

```suggestion
    EXPECT_THAT(IToA<min>::value, StrEq("-9223372036854775808"));
```